src/webgl/shader.js

```diff
@@ -86,7 +86,7 @@ export class Shader extends Resource {
         parseGLSLCompilerError(infoLog, this.source, this.shaderType);
       log.error(0, `GLSL compilation errors in ${shaderName}\n${errors}`);
       log.warn(0, `GLSL compilation warnings in ${shaderName}\n${warnings}`);
-      throw new Error(`GLSL compilation errors in ${shaderName}\n${errors}\n${warnings}`);
```

Comment:
@ibgreen ,  warnings and errors are properly identified, but the`Error` thrown here includes both warnings and errors hence we are seeing warnings as errors. 
(Note: filtered to only errors)
Before:
<img width="590" alt="screen shot 2017-09-14 at 4 07 53 pm" src="https://user-images.githubusercontent.com/9502731/30460037-2d05e632-9969-11e7-8939-fd4bafec5b19.png">
After:
<img width="547" alt="screen shot 2017-09-14 at 4 07 29 pm" src="https://user-images.githubusercontent.com/9502731/30460045-389418de-9969-11e7-81ba-606bca7c7b4c.png">

Reply:
> warnings and errors are properly identified, but theError thrown here includes both warnings and errors hence we are seeing warnings as errors.

OK that makes sense. Yes definitely let's leave the warnings out of the Exception message. Thanks for clearing it up!